pkg/forklift-api/webhooks/webhooks.go

```diff
@@ -35,8 +35,17 @@ func RegisterValidatingWebhooks() {
 		ServeSecretCreate(w, r)
 	})
 	server := http.Server{
-		Addr:    ":9876",
+		Addr:    ":8443",
 		Handler: mux,
 	}
-	server.ListenAndServeTLS("/var/run/secrets/forklift-api/tls.crt", "/var/run/secrets/forklift-api/tls.key")
+	log.Info("start listening")
+	errors := make(chan error)
+	go func() {
+		errors <- server.ListenAndServeTLS("/var/run/secrets/forklift-api-serving-cert/tls.crt", "/var/run/secrets/forklift-api-serving-cert/tls.key")
```

Comment:
I don't understand this part, why not `err = server.ListenAndServeTLS...`?

Reply:
yeah, there's no need for the go routine in this code but that's a work in progress, I think we will eventually want this code to shutdown the pod on certain signals [like kubevirt's virt-api does](https://github.com/kubevirt/kubevirt/blob/ee70cefd68aa3df0c9926e677c852e295e2229b8/pkg/virt-api/api.go#L917-L921)